_clubs/klahanie-swim-team.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Klahanie Sharks Swim Team
+date: 2019-06-10 07:36
+author: Klahanie
+comments: true
+link: true
+categories: [pools, clubs]
+show-title: true
+banner: /images/shark-swim-banner.gif
+thumbnail: /images/shark-swim-banner.gif
+description: The Klahanie Sharks are a fun swim league for kids aged 8 to 18!
+---
+The Klahanie Association is proud to partner with the the Klahanie Swim Team to provide a great opportunity for Klahanie youth to participate in team sports. The Klahanie Swim Team is a member of the 26-Team, Midlakes Summer Swim League. We seek to provide a competitive swimming environment for a young swimmer that is fun and relaxed.

```

Comment:
"We seek to provide a competitive, yet fun and relaxed, swimming environment for young swimmers."